index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:ital,wght@1,700&display=swap" rel="stylesheet">
+</head>
+<body>
+      <nav>
+          <p class="logo">My Logo</p>
+          <ul>
+            <li><a href="#">portfolio</a></li>
+            <li><a href="#">about</a></li>
+            <li><a href="#">contact</a></li>
+          </ul>
+          <div class="menu">
+            <img src="./images/Icon - Menu.png" alt="">
+            <a href="toogle-nav">Menu</a>
+          </div>
+      </nav>
+      <header>
+        <div class="topimg">
+            <div class="imgorder imgorder1">
+              <img class="backimg backimg1" src="./images/Ellipse 11.png" alt="">
+              <img class="backimg backimg2" src="./images/Rectangle 4.png" alt="">
+            </div>
+            <img src="" alt="">
+            <div class="imgorder imgorder2">
+              <img class="backimg backimg3" src="./images/Rectangle 7.png" alt="">
+              <img class="backimg backimg4" src="./images/Ellipse 5.png" alt="">
+            </div>
+            <img class="icon" src="./images/Ellipse 7.png" alt="">
+            <div class="imgorder imgorder3">
+              <img class="backimg backimg5" src="./images/Rectangle 4.png" alt="">
+              <img class="backimg backimg6" src="./images/Ellipse 3.png" alt="">
+            </div>
+        </div>
+        <h1>I’m kirubel Glad <br/> to see you!</h1>
+        <p>I’m a software developer! I can help you build a product , 
+          feature or website Look through some of my work and experience! 
+          If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p>LET’S CONNECT</p>
+        <img class="social"src="./images/social media icons.png" alt="">
+        <div class="downimg">
+             <img class="icon" src="./images/Ellipse 8.png" alt="">
+            <div class="imgorder imgorder4">
+              <img class="backimg backimg7" src="./images/Ellipse 12.png" alt="">
+              <img class="backimg backimg8" src="./images/Ellipse 13.png" alt="">
+            </div>
+            <div class="imgorder imgorder5">
+              <img class="backimg backimg9" src="./images/Ellipse 4.png" alt="">
+              <img class="backimg backimg10" src="./images/Rectangle 5.png" alt="">
+            </div>
+        </div>
+    </header>
```

Comment:
- [ ] As suggested by the previous reviewer kindly consider using the `section` tag as a parent container for the headline content.